public/app/plugins/panel/status-history/StatusHistoryPanel.tsx

```diff
@@ -196,7 +196,9 @@ export const StatusHistoryPanel = ({
   }
 
   const showNewVizTooltips =
-    config.featureToggles.newVizTooltips && (sync == null || sync() === DashboardCursorSync.Off);
+    config.featureToggles.newVizTooltips &&
+    !config.featureToggles.scenes &&
+    (sync == null || sync() === DashboardCursorSync.Off);
```

Comment:
Hm maintaining all of these checks might get a little tricky - maybe we could make a basic util function to get this state? Maybe for cursor sync panels that's a parameter we pass or something or we can just keep separate for time being as we are eventually going to support that anyways (just want to avoid having to update this in so many different spots and potentially missing something)

Reply:
I like that idea. Also, is the loose equality (==) intentional here (you may not have written this originally)? It feels a little unclear what it's trying to do. It looks like we're testing whether `sync` is either `null` or `undefined`, but using `==` instead of just deliberately checking for both feels a little cryptic.

Reply:
checking both is slower, longer, and unnecessary. it's only cryptic until you're used to it. in 99.5% of cases, null and undefined have the same meaning.